app/javascript/elm/src/Main.elm

```diff
@@ -723,7 +740,16 @@ viewDocument model =
 
 view : Model -> Html Msg
 view model =
-    div [ id "elm-app" ]
+    let
+        themeClass =
+            case Tagged.untag model.isCustomThemeOn of
```

Comment:
Have you considered a custom type for the theme? It feels like forcing a bool here. Tagged is great as a package. At the same time I feel like this is not an on / off thing. It's more like a choice. As in
```elm
type Theme
  = One -- use better name here
  | Two
```
would communicate better than `isCustomThemeOn`

Reply:
For example Michael could say: "let's switch themes". Now custom is the one that wasn't custom :D

Reply:
let's use Default and Blue, is that ok @AkkarinForest ?